src/librustc/session/config.rs

```diff
@@ -84,6 +84,7 @@ pub enum ErrorOutputType {
 }
 
 impl Default for ErrorOutputType {
+    /// Creates an `HumanReadble`, initialised with `ColorConfig` enum type `Auto`.
```

Comment:
Ditto, is there any reason to document default impls deep inside the rustc (i.e. something people reading the standard library documentation will never see)?